advanced/advanced_numpy/index.rst

```diff
@@ -249,15 +268,14 @@ dict_proxy({'block_align': (dtype('uint16'), 32), 'format': (dtype('S4'), 8), 'd
 
 >>> f = open('data/test.wav', 'r')
 >>> wav_header = np.fromfile(f, dtype=wav_header_dtype, count=1)
->>> f.close()
```

Comment:
I could also skip it.

The problem is that in Python 3 it returns the number of bytes read (and not in Python 2)